### PR TITLE
Fix some broken flags

### PR DESCRIPTION
**Pull request by mortenmj**

### CODE REVIEW

File: snapshots/go/cmd/snapshots/diff.go [hunk 1]
```diff
@@ -60,16 +62,18 @@ names.`,
 	}
 
 	// bazel flags
-	cmd.PersistentFlags().StringVar(&dc.bazelPath, "bazel-path", "", "Full URL of the storage")
-	cmd.PersistentFlags().StringVar(&dc.bazelRcPath, "bazelrc", "", "Full URL of the storage")
-	cmd.PersistentFlags().StringVar(&dc.workspacePath, "workspace-path", "", "Verbose output")
+	cmd.PersistentFlags().StringVar(&dc.bazelPath, "bazel-path", "", "full URL of the storage")
```
Comment (psalaberria002) [inline snapshots/go/cmd/snapshots/diff.go:65]: is this description correct? What kind of storage is it refering to?

**Review by psalaberria002 — APPROVED** [review]
The formatting is that a bonus feature?

File: snapshots/go/cmd/snapshots/diff.go (hunk 1 above)
Comment (mortenmj) [reply snapshots/go/cmd/snapshots/diff.go]: No, that's wrong. Fixed

**Comment by mortenmj** [comment]
> The formatting is that a bonus feature?

Which formatting do you mean?

**Comment by mortenmj** [comment]
Some things got messed up in the migration to Cobra, as snapshots no longer threw an error code back to the terminal. This caused our integration tests to succeed when they shouldn't. The first commit in this PR fixes that, and the rest fixes the problems that were caused by the migration.

**Comment by psalaberria002** [comment]
> > The formatting is that a bonus feature?
> 
> Which formatting do you mean?

format.go seems to be added i this PR.

**Review by psalaberria002 — APPROVED** [review]

**Comment by mortenmj** [comment]
> > > The formatting is that a bonus feature?
> > 
> > 
> > Which formatting do you mean?
> 
> format.go seems to be added i this PR.

Yes, that's true. That flag felt like it should be an enum, so I fixed that.

**Pull request merged**